### DISCUSSION TITLE
Tombs of Amascut, Akkah orb phase plugin

### DISCUSSION THREAD

**Discussion by Nelmo220** [runelite/runelite#15691] [Ideas]
Could we develop a plugin that if you are in the path of a traveling orb, it turns red (or whichever color you choose). An option could be added as well to select how many tiles away you want it to indicate (i.e 5 tiles away, 6 tiles, 8 tiles, etc...)

The plugin-in would NOT tell to where to stand to avoid damage, just a simple color indicator

**Comment by kamielvf** [comment]
No

> Adds additional visual or audio indicators of a boss mechanic except in cases where this is a manually triggered external helper.

[Third Party Client Guidelines](https://secure.runescape.com/m=news/third-party-client-guidelines?oldschool=1)
